### DISCUSSION TITLE
How to fully preload video and then play it in PIXI?

### DISCUSSION THREAD

**Discussion by MannySVK** [pixijs/pixijs#9275] [Q&A]
Is there any way how to first fully load video and then create Sprite with video content?

**Comment by SuperSodaSea** [comment]
Here is a similar discussion: <https://github.com/pixijs/pixijs/discussions/9212#discussioncomment-5120649>

**Reply by MannySVK** [reply]
Maybe better question would be - how to load blob data to PIXI Sprite video?

**Reply by MannySVK** [reply]
Because I have tried to load data via PIXI.Assets but larger videos are not fully loaded but only partial loaded. I mean they are being loaded even when video is being playing and that is bad in terms of performance. I want to have video fully loaded before playing.
